### PR TITLE
Extra Items 

### PR DESCRIPTION
**Pull request by satyamakgec**

### CODE REVIEW

**Review by maxsam4 — APPROVED** [review]

File: contracts/modules/STO/USDTiered/USDTieredSTOStorage.sol [hunk 1]
```diff
@@ -6,7 +6,9 @@ pragma solidity ^0.5.0;
 contract USDTieredSTOStorage {
 
     bytes32 internal constant INVESTORSKEY = 0xdf3a8dd24acdd05addfc6aeffef7574d2de3f844535ec91e8e0f3e45dba96731; //keccak256(abi.encodePacked("INVESTORS"))
-
+    bytes32 constant ADMIN = "ADMIN";
+    bytes32 constant OPERATOR = "OPERATOR";
```
Comment (tintinweb) [inline contracts/modules/STO/USDTiered/USDTieredSTOStorage.sol]: please verify if this is a duplicate declaration. `OPERATOR` and `ADMIN` are already defined in `ModuleStorage` inherited via `STO -> Module -> ModuleStorage`

File: contracts/modules/STO/Capped/CappedSTOStorage.sol [hunk 2]
```diff
@@ -5,6 +5,7 @@ pragma solidity ^0.5.0;
  */
 contract CappedSTOStorage {
 
+    bytes32 constant OPERATOR = "OPERATOR";
```
Comment (tintinweb) [inline contracts/modules/STO/Capped/CappedSTOStorage.sol]: please verify if this is a duplicate declaration. `OPERATOR` and `ADMIN` are already defined in `ModuleStorage` inherited via `STO -> Module -> ModuleStorage`

File: contracts/modules/STO/Capped/CappedSTO.sol [hunk 3]
```diff
@@ -80,8 +80,7 @@ contract CappedSTO is CappedSTOStorage, STO, ReentrancyGuard {
      * @notice Function to set allowBeneficialInvestments (allow beneficiary to be different to funder)
      * @param _allowBeneficialInvestments Boolean to allow or disallow beneficial investments
      */
-    function changeAllowBeneficialInvestments(bool _allowBeneficialInvestments) public {
-        _onlySecurityTokenOwner();
+    function changeAllowBeneficialInvestments(bool _allowBeneficialInvestments) public withPerm(OPERATOR) {
```
Comment (tintinweb) [inline contracts/modules/STO/Capped/CappedSTO.sol:83]: please check if `getPermissions()` should now return an array that contains`OPERATOR`. 
```   /**
     * @notice Return the permissions flag that are associated with STO
     */
    function getPermissions() public view returns(bytes32[] memory) {
        bytes32[] memory allPermissions = new bytes32[](0);
        return allPermissions;
    }
```

**Comment by tintinweb** [comment]
see inline comments. good to merge otherwise 👍 
tin/diligence

**Pull request merged**